app-misc/lcd4linux/lcd4linux-0.11.0_pre20170527.ebuild

```diff
@@ -0,0 +1,163 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools flag-o-matic python-single-r1
+
+DESCRIPTION="A small program that grabs information and displays it on an external LCD"
+HOMEPAGE="https://lcd4linux.bulix.org/"
+SRC_URI="https://www.bl4ckb0x.de/files/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dmalloc outb"
+REQUIRED_USE="lcd_devices_hd44780? ( !lcd_devices_hd44780-i2c )
+	lcd_devices_hd44780-i2c? ( !lcd_devices_hd44780 )
+	python? ( ${PYTHON_REQUIRED_USE} )"
```

Comment:
This is gonna make repoman unhappy.

Reply:
Are you sure? Repoman don't complain here about this?

Reply:
Sorry, my mistake. I didn't notice you're adding IUSE=python below.